src/gpu/intel/jit/conv/plan.cpp

```diff
@@ -1243,6 +1243,17 @@ std::string conv_plan_t::str() const {
     return jit::add_indent("conv_plan", oss.str());
 }
 
+type_t get_accumulation_type(
+        fma_kind_t fma_kind, const type_t &a, const type_t &b) {
```

Comment:
Comment: Would it be reasonable to standardized an accumulation getter at the primitive descriptor level? We currently reimplement the logic for the accumulation type throughout the library, which increases the likelihood of precision issues, especially as the logic around accumulation type has many details.

Reply:
This is a good idea, agree. It seems now there are three places:

- Common `convolution_desc_t`
- `jit::conv_problem_t`
- `get_accumulation_type()` in this `plan.cpp`

If you don't mind I would leave it s is in this PR and work on consolidation in [MFDNN-13271](https://jira.devtools.intel.com/browse/MFDNN-13271). Accumulation type related changes are not related to the original Jira, and the only reason I applied these changes - to work around GRF regioning limitations.